backend/tournesol/views/exports.py

```diff
@@ -84,6 +86,7 @@ def get(self, request):
 class ExportPublicComparisonsView(APIView):
     permission_classes = [AllowAny]
 
+    @method_decorator(cache_page(60*10))  # 10 minutes cache
```

Comment:
Not a problem for now, but to keep in mind:  
`cache_page` includes the current language code in the cache key. So we may need to find a workaround after multiple languages are enabled in the API (to get localized error messages, etc.) as it would be unexpected for this route to return different responses depending on the requested language.

Reply:
I'm not so worried, this endpoint is supposed to return different results when time goes on. Any difference between languages would disappear in 10 minutes. I don't think that this will ever be a source of real problems.